RadClass/RadClass.py

```diff
@@ -48,8 +48,8 @@ class RadClass:
     '''
 
     def __init__(self, stride, integration, datapath, filename, analysis=None,
-                 store_data=True, cache_size=None, start_time=None,
-                 stop_time=None,
+                 post_analysis=None, store_data=True, cache_size=None,
```

Comment:
The docstring is not up-to-date - it looks like it wasn't even before this.

Reply:
Related: As this argument list gets longer, the order of the arguments may help provide some cues to purpose.  For example, the start/stop times apply to everything, so maybe it makes sense for them to come first...